draft-ietf-tokbind-https-08.xml

```diff
@@ -486,18 +489,18 @@
 	  Token Consumer, not just the client to which it was
 	  issued.</t>
 	  <t>Therefore, in the previous step, the Token Provider may
-	  want to include in the token the Token-Binding public key that the client
+	  want to include in the token the Token Binding ID that the client
     uses when communicating with the Token Consumer, thus 
-    <spanx style="emph">binding</spanx> the token to client's Token-Binding
-    key pair. The client
+    <spanx style="emph">binding</spanx> the token to the client's Token Binding
+    private key. The client
```

Comment:
Binding the security token to the Token Binding private key sounds a little off to me. Is it bound to the private key? Or bound to the public key (TB ID)? Or bound to the key pair? I guess it's kinda bound to all those things. But using private key there jumped out at me.